AppController/djinn.rb

```diff
@@ -111,6 +111,10 @@
 APPSCALE_APP_LOGROTATE = 'appscale-app-logrotate.conf'
 
 
+# The location of the appscale-upload-app script from appscale-tools.
+UPLOAD_APP_SCRIPT = '/usr/local/bin/appscale-upload-app'
```

Comment:
Unfortunately, /usr/local/bin is not on the path when djinnServer is started from the init script. It's certainly possible for that script to be installed elsewhere, but I'm not sure the best way to handle it. We could alter the path in the init script.